### PR TITLE
Get Subchannels

### PR DESCRIPTION
**Pull request by PranavBansal21**
Issue:#56

### CODE REVIEW

File: src/controllers/subChannel.controller.js [hunk 1]
```diff
@@ -64,4 +65,20 @@ exports.deleteSubChannel = async (req, res) => {
     } catch (err) {
         return response_500(res, "Error deleting subChannel", err)
     }
-}
\ No newline at end of file
+}
+
+exports.getSubChannel = async (req, res) => {
+    try {
+      const ChannelId = req.params.id;
```
Comment (automatefreely) [inline src/controllers/subChannel.controller.js]: get channelId from req.channel._id

read issue description if any doubt

Comment (PranavBansal21) [reply src/controllers/subChannel.controller.js]: okay verify now

File: src/controllers/subChannel.controller.js [hunk 2]
```diff
@@ -64,4 +65,20 @@ exports.deleteSubChannel = async (req, res) => {
...
+}
+
+exports.getSubChannel = async (req, res) => {
+    try {
+      const ChannelId = req.channel._id;
+      const channel = await Channel.findById(ChannelId).populate("subChannels");
+      const subChannels = channel.subChannels;
+      function selectFewerProps(x) {
+        const { name, description, id } = x;
+        return { name, description, id };
```
Comment (automatefreely) [inline src/controllers/subChannel.controller.js]: id   ->     _id

Comment (PranavBansal21) [reply src/controllers/subChannel.controller.js]: only on line 77 or also on 76?

Comment (automatefreely) [reply src/controllers/subChannel.controller.js]: both

**Comment by PranavBansal21** [comment]
@automatefreely  changes done

**Pull request merged**